frontend/connection/src/constants.ts

```diff
@@ -39,8 +39,13 @@ export const HOST_CONFIG_PATH = "_stcore/host-config"
 /**
  * Min and max wait time between pings in millis.
  */
-export const PING_MINIMUM_RETRY_PERIOD_MS = 500
-export const PING_MAXIMUM_RETRY_PERIOD_MS = 1000 * 60
+export const PING_MINIMUM_RETRY_PERIOD_MS = 100
+export const PING_MAXIMUM_RETRY_PERIOD_MS = 2000
```

Comment:
I reduced these so retries happen faster. No need to wait 60s before retrying to connect -- at that point the user just refreshes the page anyway. So let's make this 2s.